### PR TITLE
[Fix] Fix loading targets in WILDRECEIPT init

### PR DESCRIPTION
**Pull request by Razlaw**
Hey :)
Thanks for the great repo.

While playing around with docTR and the Wildreceipt dataset I may have discovered a small bug.
When I print the labels in the targets for the second image, I also get the labels for the first image.

**Code to recreate bug:**
```
from doctr.datasets import WILDRECEIPT

test_set = WILDRECEIPT(train=False, img_folder=path_to_data + "/", label_path=path_to_data + "/test.txt")
img, target = test_set[1]
print(target["labels"])
```

**Result:**
`['CHOEUN', 'KOREANRESTAURANT', '2621ORANGETHORPEAVE,FULLERTON.', '(714)879-3574', 'THANKYOU!!', 'DATE', '12/30/2016FRI', '19:19', 'BIBIM.OCTOPUT1', 'S-FOODP.CAKT1', 'PORKDUMPLINT1', 'LABEEFRIBT1', '$13.99', '$14.99', '$8.99', '￥17.99', '4.00xITEMS', 'SUBTOTAL', 'TAX1', 'TOTAL', '$55.96', '$4.48', '$60.44','', '', 'TIME', "ILIO'S", 'Restaurant', 'BretonischerRing7', '85630Grasbrunn', 'TEL:', 'Steuer-Nr.:514/78510', 'RechnungNr.2844', 'Datum:', '10.07.13', '21:52', 'Tisch:102/--', '4x', '8x', '3x', '1x', '1x', '1x', '1x', '1x', '2x', '1x', '1x', '1x', '1x', '1x', 'Tee', 'Stifado', 'SchweinefiletMeta', 'BiftekiMetaxa', '', 'GyrosFolie', 'BabyKalamariGefu', 'Gyros', 'VegetarischeVaria', 'GrossesWasser', 'Saft0,25', 'Hefe-Weissbier', 'Weissbierdunkel', 'LowenbrauOriginal', 'a', 'a', 'a', 'a', '9,90', '3,30', '3,30', '3,00', '12,00', '26,40', '9,90', '2,50', '2,40', '9,90', '8,90', '12,90', '19,80', '6,90', '11,90', '13,90', '14,90', '2,10', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', 'Netto(1)', '+19,0%MwSt:', 'Eur129,75', '24,65', 'Summe:', 'EsbedienteSie:George', 'Eur154,40', '089-46169340']`

**Expected result:**
`["ILIO'S", 'Restaurant', 'BretonischerRing7', '85630Grasbrunn', 'TEL:', 'Steuer-Nr.:514/78510', 'RechnungNr.2844', 'Datum:', '10.07.13', '21:52', 'Tisch:102/--', '4x', '8x', '3x', '1x', '1x', '1x', '1x', '1x', '2x', '1x', '1x', '1x', '1x', '1x', 'Tee', 'Stifado', 'SchweinefiletMeta', 'BiftekiMetaxa', '', 'GyrosFolie', 'BabyKalamariGefu', 'Gyros', 'VegetarischeVaria', 'GrossesWasser', 'Saft0,25', 'Hefe-Weissbier', 'Weissbierdunkel', 'LowenbrauOriginal', 'a', 'a', 'a', 'a', '9,90', '3,30', '3,30', '3,00', '12,00', '26,40', '9,90', '2,50', '2,40', '9,90', '8,90', '12,90', '19,80', '6,90', '11,90', '13,90', '14,90', '2,10', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', 'Netto(1)', '+19,0%MwSt:', 'Eur129,75', '24,65', 'Summe:', 'EsbedienteSie:George', 'Eur154,40', '089-46169340']`

Could you confirm that this is a bug and not me doing something wrong? :) 

I provided a quick fix, that seems to solve the problem, but I didn't look too deep into the surrounding code.

### CODE REVIEW

**Comment by felixdittrich92** [comment]
Hi @Razlaw :wave:,

Yep you are right it's a bug thanks for the fix :+1: 

I saw that the test cases are also not matching the mocked dataset could you update this also please ?

https://github.com/mindee/doctr/blob/bdb2c2860b534db105ea01f6d3f4b8d7d3ad5956/tests/pytorch/test_datasets_pt.py#L737

--> Change from 5 to 3

https://github.com/mindee/doctr/blob/bdb2c2860b534db105ea01f6d3f4b8d7d3ad5956/tests/tensorflow/test_datasets_tf.py#L710

--> Change from 5 to 3

**Comment by Razlaw** [comment]
Sure :) Done. Thanks for the pointers.  
Btw, out of curiosity, did you use this dataset to train the models provided in docTR?

**Review by felixdittrich92 — APPROVED** [review]
Thanks @Razlaw :+1:

**Comment by felixdittrich92** [comment]
> Sure :) Done. Thanks for the pointers. Btw, out of curiosity, did you use this dataset to train the models provided in docTR?

No all our models are trained on a real mindee internal dataset (~125k samples for detection and ~10M samples for recognition) Unfortunately the dataset can't be shared because it contains sensitive data :)
